diff -Naur abf-console-client-v2.7/abf.py abf-console-client-v2.7.tpg/abf.py
--- abf-console-client-v2.7/abf.py	2018-03-01 00:40:39.530629000 +0000
+++ abf-console-client-v2.7.tpg/abf.py	2018-03-01 00:33:19.450600882 +0000
@@ -242,7 +242,7 @@
     subparser.add_argument('--testing', action='store_true', help=_('Include "testing" subrepository.'))
     subparser.add_argument('--no-extra-tests', action='store_true', help=_('Do not launch comprehensive tests.'))
     subparser.add_argument('--auto-create-container', action='store_true', help=_('enable automatic creation of container'))
-    subparser.add_argument('--cached-chroot', action='store_true', help=_('use cached chroot for the build'))
+    subparser.add_argument('--no-cached-chroot', action='store_true', help=_('do NOT use cached chroot for the build'))
     subparser.add_argument('--save-chroot', action='store_true', help=_('save build chroot in case of failure'))
     subparser.add_argument('--update-type', action='store', choices=BuildList.update_types, help=_('Update type. Default is "%s".') %
                     (BuildList.update_types[0]) )
@@ -286,7 +286,7 @@
     subparser.add_argument('--testing', action='store_true', help=_('Include "testing" subrepository.'))
     subparser.add_argument('--no-extra-tests', action='store_true', help=_('Do not launch comprehensive tests.'))
     subparser.add_argument('--auto-create-container', action='store_true', help=_('enable automatic creation of container'))
-    subparser.add_argument('--cached-chroot', action='store_true', help=_('use cached chroot for the build'))
+    subparser.add_argument('--no-cached-chroot', action='store_true', help=_('do NOT use cached chroot for the build'))
     subparser.add_argument('--save-chroot', action='store_true', help=_('save build chroot in case of failure'))
     subparser.add_argument('--update-type', action='store', choices=BuildList.update_types, help=_('Update type. Default is "%s".') %
                     (BuildList.update_types[0]) )
@@ -1431,10 +1431,15 @@
     if auto_create_container is None:
         auto_create_container = True
 
-    if command_line.no_extra_tests is None:
-        use_extra_tests = True
+    if command_line.no_cached_chroot:
+        cached_chroot = False
     else:
+        cached_chroot = True
+
+    if command_line.no_extra_tests:
         use_extra_tests = False
+    else:
+        use_extra_tests = True
 
     if not command_line.auto_publish and not command_line.auto_publish_status:
         command_line.auto_publish_status = default_publish_status
@@ -1458,7 +1463,7 @@
         command_line.auto_publish_status or BuildList.auto_publish_statuses[0],
         arches,
         command_line.skip_personal,
-        command_line.cached_chroot,
+        cached_chroot,
         command_line.save_chroot,
         auto_create_container,
         command_line.testing,
diff -Naur abf-console-client-v2.7/bash_autocomplete abf-console-client-v2.7.tpg/bash_autocomplete
--- abf-console-client-v2.7/bash_autocomplete	2017-08-21 15:29:46.000000000 +0000
+++ abf-console-client-v2.7.tpg/bash_autocomplete	2018-03-01 00:38:48.545628090 +0000
@@ -69,7 +69,7 @@
 
 __abf_build()
 {
-  __abf_opts "--branch --build-list --tag --commit --target-platform --arch --repository --save-to-repository --auto-publish-status --auto-publish --update-type --skip-spec-check --auto-create-container --no-extra-tests --cached-chroot --save-chroot --testing --external-nodes"
+  __abf_opts "--branch --build-list --tag --commit --target-platform --arch --repository --save-to-repository --auto-publish-status --auto-publish --update-type --skip-spec-check --auto-create-container --no-extra-tests --no-cached-chroot --save-chroot --testing --external-nodes"
   update_types="security bugfix enhancement recommended newpackage"
   external_nodes_vals="owned everything"
   auto_publish_status="none default testing"
@@ -116,7 +116,7 @@
 
 __abf_chain_build()
 {
-  __abf_opts "--branch --build-list --tag --infile --commit --target-platform --arch --repository --save-to-repository --auto-publish-status --auto-publish --skip-spec-check --auto-create-container --no-extra-tests --cached-chroot --testing"
+  __abf_opts "--branch --build-list --tag --infile --commit --target-platform --arch --repository --save-to-repository --auto-publish-status --auto-publish --skip-spec-check --auto-create-container --no-extra-tests --no-cached-chroot --testing"
   auto_publish_status="none default testing"
 
   if [ ${prev} == --auto-publish-status ] ; then
diff -Naur abf-console-client-v2.7/po/abf-console-client.pot abf-console-client-v2.7.tpg/po/abf-console-client.pot
--- abf-console-client-v2.7/po/abf-console-client.pot	2017-08-21 15:29:46.000000000 +0000
+++ abf-console-client-v2.7.tpg/po/abf-console-client.pot	2018-03-01 00:39:38.029629037 +0000
@@ -513,7 +513,7 @@
 msgstr ""
 
 #: ../abf.py:245 ../abf.py:289
-msgid "use cached chroot for the build"
+msgid "do NOT use cached chroot for the build"
 msgstr ""
 
 #: ../abf.py:246 ../abf.py:290
diff -Naur abf-console-client-v2.7/po/ru.po abf-console-client-v2.7.tpg/po/ru.po
--- abf-console-client-v2.7/po/ru.po	2017-08-21 15:29:46.000000000 +0000
+++ abf-console-client-v2.7.tpg/po/ru.po	2018-03-01 00:40:23.994629366 +0000
@@ -604,8 +604,8 @@
 msgstr "включить автоматическое создание контейнера"
 
 #: ../abf.py:245 ../abf.py:289
-msgid "use cached chroot for the build"
-msgstr "использовать для сборки кэшированное окружение"
+msgid "do NOT use cached chroot for the build"
+msgstr "не используйте кэшированную среду для сборки"
 
 #: ../abf.py:246 ../abf.py:290
 msgid "save build chroot in case of failure"
