diff -Naur abf-console-client-v2.7/abf.py abf-console-client-v2.7.tpg/abf.py
--- abf-console-client-v2.7/abf.py	2018-03-01 00:42:21.799628000 +0000
+++ abf-console-client-v2.7.tpg/abf.py	2018-03-01 00:43:14.279626682 +0000
@@ -1416,8 +1416,11 @@
             arches.append(a)
     else:
         try_arches = ['i586','i686','x86_64']
-        if str(build_repositories[0]).find("openmandriva") >= 0 or str(build_repositories[0]).find("cooker") >= 0:
-            try_arches = ['i586','i686','x86_64','armv7hl','aarch64']
+        if str(build_repositories[0]).find("openmandriva") >= 0 or str(build_repositories[0]).find("3.0") >= 0:
+            try_arches = ['i586','x86_64','armv7hnl','aarch64']
+
+        if str(build_repositories[0]).find("cooker") >= 0 or str(build_repositories[0]).find("4.0") >= 0:
+            try_arches = ['i686','x86_64','armv7hnl','aarch64']
 
         for arch in try_arches:
             a = Arch.get_arch_by_name(models, arch)
