--- abf-console-client-v2.5.3/abf/console/config.py.orig	2016-03-12 15:58:16.794964109 +0000
+++ abf-console-client-v2.5.3/abf/console/config.py	2016-03-12 15:58:43.897962376 +0000
@@ -94,8 +94,8 @@
         return res
 
 class Config(dict):
-    default_url = 'https://abf.rosalinux.ru'
-    default_filestore_url = 'http://file-store.rosalinux.ru'
+    default_url = 'https://abf.openmandriva.org'
+    default_filestore_url = 'http://file-store.openmandriva.org'
     default_log_path = '/var/log/abf.log'
     def __init__(self, conf_path='~/.abfcfg', main_conf=True):
         self.conf_path = os.path.expanduser(conf_path)
@@ -175,8 +175,9 @@
             self['user']['password'] = password
 
         parts = self['main']['abf_url'].split('//')
-        git_uri = "%(protocol)s//%(user)s@%(domain)s" % \
-                dict(protocol=parts[0], user=self['user']['login'], domain=parts[1])
+        #git_uri = "%(protocol)s//%(user)s@%(domain)s" % \
+        #        dict(protocol=parts[0], user=self['user']['login'], domain=parts[1])
+        git_uri = "ssh://git@github.com/OpenMandrivaAssociation"
 
         self['user']['git_uri'] = git_uri
 
