diff -Naur abf-console-client-v2.7/abf/console/misc.py abf-console-client-v2.7.tpg/abf/console/misc.py
--- abf-console-client-v2.7/abf/console/misc.py	2017-08-21 15:29:46.000000000 +0000
+++ abf-console-client-v2.7.tpg/abf/console/misc.py	2017-09-07 18:35:30.723077694 +0000
@@ -45,10 +45,11 @@
         e["LC_ALL"] = "C"
         output, ret_code = execute_command(['git', 'remote', 'show', 'origin', '-n'], cwd=path, env=e)
 
-        m = re.compile("^.*Fetch URL:\s+.*abf.*:(.+)/([^/]+)[.]git$",re.MULTILINE).search(output)
+        m = re.compile("^.*Fetch URL:\s+.*OpenMandrivaAssociation.*:(.+)/([^/]+)[.]git$",re.MULTILINE).search(output)
         if m:
+            # Cant tell abf owner from github loc
             owner_name, project_name = m.groups()
-            return (owner_name, project_name)
+            return ('openmandriva', project_name)
         return (None, None)
     except ReturnCodeNotZero:
         return (None, None)
diff -Naur abf-console-client-v2.7/abf.py abf-console-client-v2.7.tpg/abf.py
--- abf-console-client-v2.7/abf.py	2017-09-07 18:34:39.019075000 +0000
+++ abf-console-client-v2.7.tpg/abf.py	2017-09-07 18:34:58.730076475 +0000
@@ -797,7 +797,7 @@
     elif len(tmp) == 2:
         project_name = tmp[1]
 
-    uri = "%s/%s.git" % (cfg['user']['git_uri'], proj)
+    uri = "%s/%s.git" % (cfg['user']['git_uri'], project_name)
     cmd = ['git', 'clone', uri]
     if command_line.branch:
         cmd += ['-b', command_line.branch]
@@ -1288,7 +1288,7 @@
     # get git commit hash
 
     commit_hash = None
-    if tmp == 0:
+    if tmp == 0 and (not as_branch or not as_saveto):
         log.error(_("Git branch, tag or commit can not be resolved automatically. Specify it by -b, -t or -c."))
         exit(1)
     if tmp == 1:
